src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,96 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+import { evaluateOnce } from '../derived/evaluateOnce';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  let returnMethod;
+  const newValue = new Value(0);
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  currentState.position = newValue;
+  let alwaysNode;
+  let isStarted = false;
+  let isDone = false;
+  let wasStopped = false;
+  return {
+    start: currentReturnMethod => {
+      if (isStarted) {
+        returnMethod && returnMethod({ finished: false });
+        return;
+      }
+      if (isDone) {
+        console.warn('Animation has been finished before');
```

Comment:
how is this different comparing to what we check above? If `isStarted` is true and we start again shouldn't we warn too? Instead we call the finished callback – is that consistent?